lib/msf/core/post/linux/packages.rb

```diff
@@ -0,0 +1,74 @@
+# -*- coding: binary -*-
+
+module Msf
+  class Post
+    module Linux
+      module Packages
+        include ::Msf::Post::Linux::System
+
+        #
+        # Determines the version of an installed package
+        #
+        # @param package The package name to check for
+        # @return [Rex::Version] nil if OS is not supported or package is not installed
+        #
+        def installed_package_version(package)
+          info = get_sysinfo
+
+          if ['debian', 'ubuntu'].include?info[:distro]
```

Comment:
Just for consistency:
```suggestion
          if ['debian', 'ubuntu'].include?(info[:distro])
```